src/WinUIEx/HwndExtensions.cs

```diff
@@ -115,6 +115,131 @@ public static void CenterOnScreen(IntPtr hwnd, double? width = null, double? hei
             SetWindowPosOrThrow(new HWND(hwnd), new HWND(), left, top, w, h, SET_WINDOW_POS_FLAGS.SWP_SHOWWINDOW);
         }
 
+        /// <summary>
+        /// Moves the window to the specified coordinates
+        /// </summary>
+        /// <param name="hwnd">Window handle</param>
+        /// <param name="x">Left side of the window</param>
+        /// <param name="y">Top side of the window</param>
+        public static void MoveToPoint(IntPtr hwnd, double x, double y)
+        {
+            PInvoke.GetWindowRect(new HWND(hwnd), out RECT windowRect);
+            var w = windowRect.right - windowRect.left;
+            var h = windowRect.bottom - windowRect.top;
+            var left = x;
+            var top = y;
+            SetWindowPosOrThrow(new HWND(hwnd), new HWND(), (int)left, (int)top, w, h, SET_WINDOW_POS_FLAGS.SWP_SHOWWINDOW);
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="hwnd"></param>
+        /// <param name="x"></param>
+        /// <param name="y"></param>
+        /// <param name="width"></param>
+        /// <param name="height"></param>
+        public static void CenterOnPoint(IntPtr hwnd, double x, double y, double? width = null, double? height = null)
+        {
+            var dpi = PInvoke.GetDpiForWindow(new HWND(hwnd));
+            PInvoke.GetWindowRect(new HWND(hwnd), out RECT windowRect);
+            var scalingFactor = dpi / 96d;
+            var w = width.HasValue ? (int)(width * scalingFactor) : windowRect.right - windowRect.left;
+            var h = height.HasValue ? (int)(height * scalingFactor) : windowRect.bottom - windowRect.top;
+            var cx = x;
+            var cy = y;
+            var left = cx - (w / 2);
+            var top = cy - (h / 2);
+            SetWindowPosOrThrow(new HWND(hwnd), new HWND(), (int)left, (int)top, w, h, SET_WINDOW_POS_FLAGS.SWP_SHOWWINDOW);
+        }
+
+        /// <summary>
+        /// The edge positions MoveToPosition can move to
+        /// </summary>
+        public enum Positions
+        {
+            TopLeft,
+            TopCenter,
+            TopRight,
+            MiddleLeft,
+            MiddleCenter,
+            MiddleRight,
+            BottomLeft,
+            BottomCenter,
+            BottomRight
+
+        }
```

Comment:
This should probably be in the WinUIEx namespace instead of inside a class - especially considering it is used in multiple places and not just in this class. Also this should generate warnings as there is no summary on each value.
Lastly can we come up with a better name for this enum? (also shouldn't be plural)

Reply:
Good idea, I'll move it.
WindowPosition? Position?

Reply:
How about just “Alignment” or “Placement”